akvo/rsr/spa/app/components/StatusPeriod.jsx

```diff
@@ -1,14 +1,17 @@
 import React from 'react'
 import SVGInline from 'react-svg-inline'
 import { Button } from 'antd'
+import { useTranslation } from 'react-i18next'
+
 import approvedSvg from '../images/status-approved.svg'
 import pendingSvg from '../images/status-pending.svg'
 import revisionSvg from '../images/status-revision.svg'
 import { DeclinePopup } from './DeclinePopup'
 
 const Aux = node => node.children
 
-export const StatusPeriod = ({ update, pinned, index, handleUpdateStatus, t }) => {
+export const StatusPeriod = ({ update, pinned, index, handleUpdateStatus }) => {
+  const { t } = useTranslation()
```

Comment:
It's better to have descriptive name instead of `t`

Reply:
Noted, Thank you Navin. Fixing in progress